app/tests/mocks/window.js

```diff
@@ -103,6 +103,7 @@ define(function (require, exports, module) {
     this.sessionStorage = new NullStorage();
     this.top = this;
 
+    this.$ = window.$;
```

Comment:
We have been trying to not access `$` directly on the window, instead using `var $ = require('jquery')`, is this necessary?